changelog/4709.doc.rst

```diff
@@ -0,0 +1 @@
+Created the new gallery example :ref:`sphx-glr-generated-gallery-units-and-coordinates-north-offset-frame-py` for `~sunpy.coordinates.NorthOffsetFrame`.
```

Comment:
```suggestion
Created the new gallery example :ref:`sphx_glr_generated_gallery_units_and_coordinates_north_offset_frame_py` for `~sunpy.coordinates.NorthOffsetFrame`.
```
Oops, I mistakenly suggested hyphens instead of underscores